.circleci/config.yml

```diff
@@ -331,6 +370,36 @@ jobs:
               $host.SetShouldExit(1)
             }
 
+  merge-release-branch:
+    parameters:
+      e:
+        type: executor
+    executor: << parameters.e >>
+    steps:
+      - slack/notify:
+          channel: 'pdt_releases'
+          color: '#FFDEAD'
+          message: 'Merging release branch into main'
+      - ssh-config
+      - checkout
+      - gh-config
+      - prepare-release-environment-variables
+      - merge-validation
+      - run:
+          name: 'Rebasing main off of release branch'
+          command: |
+            echo "Rebasing main off of release branch ${RELEASE_VERSION}"
+            git checkout main
+            git rebase -Xtheirs release/v${RELEASE_VERSION}
```

Comment:
Originally I had this command as `git rebase release/v${RELEASE_VERSION}`. But I was able to reproduce a conflict that was causing issues yesterday evening. This appears to work, though I'm wondering if we want to handle conflicts manually for now - so we understand what is happening behind the scenes. I haven't been able to reproduce the conflict again, so I'm not 100% sure this is the resolution we need.

Reply:
Also to address a potential question - why rebase vs. merge?

With rebase we are ensuring that all new commits are rewriting the commit history for the main branch. Everything specific to the release will be on "top" of everything in main. This will keep the history linear to reflect our releases.

I did also explore using "merge" as an option here. However, when it comes to this specific step it seems like there's a problem with the actual merge commit message that follows. There's no issue with merging main into develop (likely because of the rebase that happens here), it's specific to this phase of the process.

Example Error that was Seen:
`Auto-merging README.md
error: Terminal is dumb, but EDITOR unset
Not committing merge; use 'git commit' to complete the merge.`

There is likely a good way to resolve this if we still want to consider using merge instead of rebase here to be consistent. Since the rebase seems to work more reliably, I decided to just continue with what worked best.